app/controllers/api/v1/configuration_profiles_controller.rb

```diff
@@ -13,19 +13,25 @@ def create
       end
 
       def index
```

Comment:
If I'm not mistaken, this action is only used to fetch configuration profiles for admins in the dashboard. If so, granting access to non-admins and even more so non-authenticated users is not quite good.

Reply:
thanks for catching, will push update on that tomorrow

Reply:
@excelsior pushed updates on this.